resources/sdk/clisdkclient/config.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "GO CLI SDK",
+  "envVars": {
+  },
+  "settings": {
+    "sdkRepo": {
+      "repo": "https://github.com/ronanwatkins/platform-client-sdk-go.git",
```

Comment:
Will need to update this to the repo we're going to use for this SDK

Reply:
https://github.com/MyPureCloud/platform-client-sdk-cli

Repo is created, but I haven't added it to the access control list. Go ahead and add it and add yourself as a maintainer so you can do the initial setup in the repo.